python/lsst/sims/maf/metrics/calibrationMetrics.py

```diff
@@ -121,6 +121,7 @@ def __init__(self, metricName='properMotion',
         super(ProperMotionMetric, self).__init__(col=cols, metricName=metricName, units=units,
                                                  badval=badval, **kwargs)
         # set return type
+        self.mjdCol = mjdCol
```

Comment:
Seeing this added here made me have a look at the Parallax Metric .. it asks for the MJDcol in the init but doesn't use it, instead relying on the stacker to add ra_pi_amp and dec_pi_amp. How does that work in this situation? 
And can you drop the extra (but not used) columns from ParallaxMetric?